File: test/python/pokemon-color.py
import unittest
import pokeapiclient
from pokeapiclient.rest import ApiException
from pprint import pprint

class TestPokemonColor(unittest.TestCase):

    def test_pokemon_color_retrieve(self):

      configuration = pokeapiclient.Configuration(
          host = "https://pokeapi.co"
      )

      with pokeapiclient.ApiClient(configuration) as api_client:

          api_instance = pokeapiclient.PokemonApi(api_client)

          try:
              api_response = api_instance.pokemon_color_retrieve('black')
              print("The response of PokemonApi->pokemon_color_retrieve:\n")
              pprint(api_response)
              self.assertIsInstance(api_response, pokeapiclient.PokemonColorDetail)
          except ApiException as e:
              self.fail('Exception when calling PokemonApi->pokemon_color_retrieve: %s\n' % e)

File: test/javascript/pokemon-color.js
const assert = require('assert');
const PokeapiClient = require('pokeapi-client');

describe('pokemon-color', function() {
  describe('retrieve', function() {
    it('should respond with pokemon color detail', function(done) {
      var callback = function(error, data, response) {
        assert.equal(error, null);
        console.dir(data);
        assert.equal(data instanceof PokeapiClient.PokemonColorDetail, true);
        assert.equal(response.res.statusCode, 200);
        done();
      };

      const apiClient = new PokeapiClient.PokemonApi();
      apiClient.pokemonColorRetrieve('black', callback);
    });
  });
});